New Field in UI - default release channel (#2562)
kyma-project/kyma-environment-broker: notable-changes/1.25.0/notable-change.md

<!--{"metadata":{"requirement":"MANDATORY","type":"EXTERNAL","category":"FEATURE","additionalFiles":0}}-->

# Updating Kyma Environment Broker

> [!WARNING]

> > This update is mandatory. Without performing it, you will not be able to use the feature in SAP BTP cockpit.

Reviewer: ```suggestion
> This update is mandatory. Without performing it, you will not be able to use the feature in the SAP BTP cockpit.
```

## Prerequisites

- Access to the ERS registry.

## What's Changed

> - It is possible to set preffered **Default Release Channel**.

Reviewer: ```suggestion
You can set your preferred **Default Release Channel**.
```
